src/python/WMComponent/AgentStatusWatcher/AgentStatusPoller.py

```diff
@@ -106,6 +107,12 @@ def algorithm(self, parameters):
         get information from wmbs, workqueue and local couch
         """
         try:
+            config_path = os.path.join(os.environ.get('config', ''), 'config.py')
```

Comment:
Alex, can you also rename this env var to WMA_CONFIG please? We'll have to update the env script `/data/admin/wmagent/env.sh`.